### PR TITLE
Refactor CSS module

### PR DESCRIPTION
**Pull request by blinker1104**
html runs okay without css

### CODE REVIEW

File: client/components/UiButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import UIReviewButton from './UiReviewButton.jsx';
+// import UIReviewButton from './UiReviewButton.jsx';//client\components\overview_ui\UiReviewButton.jsx
```
Comment (nathanksou) [inline client/components/UiButtons.jsx:3]: Just making it aware this is here.

File: client/components/UiShareButton.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+class UIShareButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showShare: '',
+      showOverlay: ''
+    };
+  }
+
+  componentDidMount () {
```
Comment (nathanksou) [inline client/components/UiShareButton.jsx:14]: Does this need to exist if it is not being used? Or will you use it later?

File: server/config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+const url = 'http://localhost:3001';
```
Comment (nathanksou) [inline server/config.js:2]: Not sure if you meant to gitignore config.js.

**Review by nathanksou — APPROVED** [review]
A few notes on this commit. Overall changes look amazing! Great work on revising to ES6 syntax and CSS Modules!

File: server/config.js (hunk 3 above)
Comment (blinker1104) [reply server/config.js:2]: it is not doing anything actually, planning to remove or change.

**Comment by blinker1104** [comment]
I accidentally hit merge before commit...
I already made a changes and try to update the changes in next pull request.

**Pull request merged**